Blankmon/src/application/BaseAI.java

```diff
@@ -0,0 +1,63 @@
+package application;
+import java.util.ArrayList;
+import java.util.Random;
+import application.enums.*;
+
+public class BaseAI {
+	public BaseAI() {
+		super(); 
+	}
+	
+	private final String useItem(ArrayList<Item> items, Anature currentAnature, int healthThreshold) {
+		if(items.isEmpty()) {
+			return "Cannot use Item because there are none";
+		} else if(currentAnature.getHp() < healthThreshold) {
+			return "Item Consumed";
+		} else {
+			return "";
+		}
+		
+		 
+	}
+	
+	public final String switchAnature(ArrayList<Anature> anatures, Type type, int switchThreshold ) {
+		if(!anatures.isEmpty() && isAnatureAtTypeDisadvantage(type) && willSwitch(switchThreshold)) {
+			return "Switch Anature"; 
```

Comment:
We also want this to be a hook method so that, when the Trainer's Anature dies, they can summon a new one without having to go through the logic checks. Also we need to have a parameter saying which anature is being used so that we don't accidentally switch into the same anature.